apps/dashboard/src/app/knowledge-box/utils.ts

```diff
@@ -7,3 +9,36 @@ export const KB_ROLE_TITLES: { [role in KBRoles]: string } = {
   SCONTRIBUTOR: 'generic.contributor',
   SMEMBER: 'generic.member',
 };
+
+export function getDesktopPlatform(): 'win' | 'mac' | 'linux' | null {
+  const platform = window.navigator.platform.toLowerCase();
+  if (platform.includes('win')) {
+    return 'win';
+  } else if (platform.includes('mac')) {
+    return 'mac';
+  } else if (platform.includes('linux')) {
+    return 'linux';
+  }
+  return null;
+}
+
+export function getDesktopAppUrl(platform: 'mac' | 'win' | 'linux'): Observable<string | null> {
+  const filenames = {
+    mac: /.*-universal-mac\.zip$/,
+    win: /.*\.exe$/,
+    linux: /.*\.snap$/,
+  };
+  return fromFetch('https://api.github.com/repos/nuclia/frontend/releases/latest').pipe(
```

Comment:
you're using this URL here and on `knowledge-box-home.component.ts` as well, you could export it in a const in utils.ts, and use this const everywhere instead.

Reply:
fixed